Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -511,6 +511,20 @@ class ChangeToChinaSetting: HiddenSetting {
     }
 }
 
+class FasterInactiveTabs: HiddenSetting {
+    override var title: NSAttributedString? {
+        let isFasterEnabled = UserDefaults.standard.bool(forKey: PrefsKeys.FasterInactiveTabsOverride)
+        let buttonTitle = isFasterEnabled ? "Debug: Inactive Tab Timeout 10s" : "Debug: Inactive Tab Timeout Default"
```

Comment:
```suggestion
        let buttonTitle = isFasterEnabled ? "Debug: Set Inactive Tab Timeout to Default" : "Debug: Set Inactive Tab Timeout to 10s"
```
For Debug stuff, we should say what tapping the action does, rather than the current state, for clarity. That's the pattern we've adopted for new (or refactored) debug menu items.

Reply:
Changed the row title and run lint